tlatools/org.lamport.tlatools/test-model/Test3.tla

```diff
@@ -16,8 +16,7 @@ Prop1 ==  <>[](x#1) \/ <>[](x#2)
 
 PostCondition ==
 	/\ TLCSet(42, TLCGet("generated"))
-	/\ \/ CounterExample = [state|->{},action|->{}] 
-	   \/ /\ ToTrace(CounterExample) = <<[x |-> 0], [x |-> 1], [x |-> 0], [x |-> 2]>>
+	/\ \/ /\ ToTrace(CounterExample) = <<[x |-> 0], [x |-> 1], [x |-> 0], [x |-> 2]>>
```

Comment:
Given that some tests started failing, does it mean that some user's spec might start failing too after merging this change? Also, why are these tests failing now?

Reply:
```tla
\/ CounterExample = [state|->{},action|->{}] 
	   \/
```

Back then, I added this disjunct to make the test assertions more liberal. This was done to ensure the tests would pass even when the POSTCONDITION was evaluated twice. :-p